docs/ref/databases.txt

```diff
@@ -584,10 +584,12 @@ Notes on specific fields
 Character fields
 ~~~~~~~~~~~~~~~~
 
-Any fields that are stored with ``VARCHAR`` column types have their
+Any fields that are stored with ``VARCHAR`` column types may have their
 ``max_length`` restricted to 255 characters if you are using ``unique=True``
 for the field. This affects :class:`~django.db.models.CharField`,
-:class:`~django.db.models.SlugField`.
+:class:`~django.db.models.SlugField`. Refer to `mysql documentation`_ for details.
+
+.. _mysql documentation: https://dev.mysql.com/doc/refman/8.0/en/char.html
```

Comment:
Links should not include a specific version, use `https://dev.mysql.com/doc/refman/en/char.html`, also I'm not sure if that's the best link to describe this issue. @adamchainz Do you have a recommendation?

Reply:
@felixxm @adamalton Any suggestions for the best link to describe this issue.

Reply:
I think I've been Cc'd on this by mistake, but now that I'm here 🙂… I had a bit of a dig around in the MySQL docs, and I can't actually find anything that says that the length of a unique index on a VARCHAR column is limited to 255 characters. Is it even true?

This was the most relevant page I could find: https://dev.mysql.com/doc/refman/8.0/en/storage-requirements.html#data-types-storage-reqs-strings

Other relevant pages are:  
https://dev.mysql.com/doc/refman/8.0/en/create-index.html#create-index-column-prefixes and futher down that page:  
https://dev.mysql.com/doc/refman/8.0/en/create-index.html#create-index-unique

But none of them mention a limit of 255 characters 🤷‍♂

Reply:
@adamalton   I am sorry for ccing you. This issue is regarding Django itself limiting the varchar to 255 chars.

Reply:
Thank you Mr Alton. The [column prefix key parts](https://dev.mysql.com/doc/refman/8.0/en/create-index.html#create-index-column-prefixes) link is a relevant reference. It mentions 767 bytes = 255 characters using the 3 byte characters in the default utf8 (utf8mb3) character set.

We can't use the "dev" link as @felixxm says since it appears the url is a 404 there.

Reply:
> We can't use the "dev" link as @felixxm says since it appears the url is a 404 there.

Both "dev" links redirect properly to the current version for me :thinking: 
- https://dev.mysql.com/doc/refman/en/char.html
- https://dev.mysql.com/doc/refman/en/create-index.html#create-index-column-prefixes

Reply:
Oh my bad, I edited the URL incorrectly. You're right, the last one is the right URL there.